source/common/secret/secret_manager_impl.h

```diff
@@ -14,8 +14,9 @@
 namespace Envoy {
 namespace Secret {
 
-class SecretManagerImpl : public SecretManager {
+class SecretManagerImpl : public SecretManager, public Logger::Loggable<Logger::Id::secret> {
```

Comment:
Do we need this or is it a left-over from debugging?

Reply:
Removed.